modules/cli-installing-cli-web-console.adoc

```diff
@@ -0,0 +1,121 @@
+ifeval::["{context}" == "updating-restricted-network-cluster"]
+:restricted:
+endif::[]
+
+[id="cli-installing-cli-web-console_{context}"]
+= Installing the OpenShift CLI by using the web console
+
+You can install the OpenShift CLI (`oc`) to interact with {product-title} from a web console. You can install `oc` on Linux, Windows, or macOS.
+
+[IMPORTANT]
+====
+If you installed an earlier version of `oc`, you cannot use it to complete all
+of the commands in {product-title} {product-version}. Download and
+install the new version of `oc`.
+ifdef::restricted[]
+If you are upgrading a cluster in a restricted network, install the `oc` version that you plan to upgrade to.
+endif::restricted[]
+====
+
+[discrete]
+== Installing the OpenShift CLI on Linux using the web console
+
+You can install the OpenShift CLI (`oc`) binary on Linux by using the following procedure.
+
+.Procedure
+
+. From the web console, click *?*.
++
+image::click-question-mark.png[]
+. Click *Command Line Tools*.
++
+image::CLI-list.png[]
+. Select appropriate `oc` binary for your Linux platform and click *Download oc for Linux*.
```

Comment:
Change to: "...platform, and then click..."

Reply:
acked